app/views/layouts/application.html.erb

```diff
@@ -80,5 +78,6 @@
         </div>
       </div>
     </footer>
+    <%= javascript_include_tag :application %>
```

Comment:
What's your reasoning to moving the javascript tag to the bottom of the page?

If it's for performance, I believe this strategy is outdated, and that using the [`async`](https://kapeli.com/dash_share?docset_file=HTML&docset_name=HTML&path=developer.mozilla.org/en-US/docs/Web/HTML/Element/script.html%23attr-async&platform=html&repo=Main&source=developer.mozilla.org/en-US/docs/Web/HTML/Element/script) attribute and including it in the `head` is the preferred approach.

If it's to ensure the HTML elements are on the page, before the script is executed, then I believe using a `$(document).ready` handler is more appropriate, as it allows the Javascript to be fetched, parsed, and executed in parallel with parsing and laying out the page.